packages/plugin-rag/src/rag.worker.ts

```
@@ -0,0 +1,487 @@
+          );
+        }
+        await processAndChunkDocument({
+          documentId: message.payload.documentId,
+          fileContentB64: message.payload.fileContentB64,
+          contentType: message.payload.contentType,
+          originalFilename: message.payload.originalFilename || 'unknown_file',
+        });
+        break;
+      case 'PROCESS_PDF_THEN_FRAGMENTS':
+        const { clientDocumentId, fileContentB64, contentType, originalFilename, worldId } =
+          message.payload;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)